[quant][graphmode][fix] docs for InsertObservers

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #35721 [quant][graphmode][refactor] Move the whitelists to a centeralized place
* #35688 [quant] Move quantization tests into test/quantization
* #35571 [quant][graphmode] Quantization support for at::sort
* #35558 [quant][graphmode][refactor] getGeneralTensorInputs(Node*) -> getPassThroughInputs(Value*)
* **#35557 [quant][graphmode][fix] docs for InsertObservers**

Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

Differential Revision: [D20752129](https://our.internmc.facebook.com/intern/diff/D20752129)

jerryzh168 committed:
[quant][graphmode][fix] docs for InsertObservers

Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar commented on torch/csrc/jit/passes/quantization.cpp:
```diff
@@ -612,16 +612,18 @@ class InsertObserversHelper {
+  std::unordered_map<Value*, std::unordered_set<Value*>> boundary_value_map_;
+  std::unordered_set<Value*> observed_values_;
+  // This is used for the observed values to pass through the ops like flatten,
+  // so that output value of flatten do not need to be observed
```
nit: does not need to be observed.

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] docs for InsertObservers"


Summary:
.

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]
